app/frontend/src/components/Button/Button.tsx

```diff
@@ -12,7 +12,7 @@ import CircularProgress from "../CircularProgress";
 
 const useStyles = makeStyles((theme) => ({
   contained: {
-    borderRadius: `${theme.shape.borderRadius * 2}px`,
+    borderRadius: theme.spacing(0.5),
```

Comment:
Why this change? Should border radius be set in relation to the theme border radius?

Reply:
The border radius was too high compared to the design, and I didn't want to hack around the theme's border radius with some weird maths. Maybe we should lower the value so it's more flexible, although that might involve touching some unrelated file in this PR if I do that...